xcube/core/dsio.py

```diff
@@ -530,6 +552,7 @@ def write(self, dataset: xr.Dataset, output_path: str, **kwargs):
         dataset.to_dataframe().to_csv(output_path, **kwargs)
 
 
+@deprecated(_DEPRECATION_REASON, version=_DEPRECATION_VERSION)
 def rimraf(*paths: str):
```

Comment:
This is the one method that could actually survive. We should just move it to some other place. Somewhere in util maybe.